package.json

```diff
@@ -31,20 +31,22 @@
     "test": "yarn lint --quiet && yarn unit && yarn type-check && yarn diff:sample-json",
     "test-extension": "cd lighthouse-extension && yarn test",
     "test-viewer": "cd lighthouse-viewer && yarn pptr-test",
-
-    "unit-core": "mocha --reporter dot \"lighthouse-core/test/**/*-test.js\"",
-    "unit-cli": "mocha --reporter dot \"lighthouse-cli/test/**/*-test.js\"",
+    "unit-core": "jest \"lighthouse-core/\"",
+    "unit-core:ci": "jest --runInBand --coverage --ci \"lighthouse-core/\"",
+    "unit-cli": "jest  --runInBand \"lighthouse-cli/\"",
+    "unit-cli:ci": "jest --runInBand --coverage --ci \"lighthouse-cli/\"",
     "unit-viewer": "mocha --reporter dot \"lighthouse-viewer/test/**/*-test.js\"",
     "unit": "yarn unit-core && yarn unit-cli && yarn unit-viewer",
+    "unit:ci": "yarn unit-core:ci && yarn unit-cli:ci && yarn unit-viewer",
     "core-unit": "yarn unit-core",
     "cli-unit": "yarn unit-cli",
     "viewer-unit": "yarn unit-viewer",
     "watch": "yarn unit-core --watch",
 
-    "unit:silentcoverage": "nyc --silent yarn unit && nyc report --reporter text-lcov > unit-coverage.lcov",
-    "coverage": "nyc yarn unit && nyc report --reporter html",
-    "smoke:silentcoverage": "nyc --silent yarn smoke && nyc report --reporter text-lcov > smoke-coverage.lcov",
-    "coverage:smoke": "nyc yarn smoke && nyc report --reporter html",
+    "unit:silentcoverage": "nyc --silent --clean yarn unit:ci && nyc report --reporter text-lcov > unit-coverage.lcov",
```

Comment:
one unfortunate side of this (at least judging from that comment in the jest issue) is that subprocesses don't automatically collect coverage. I don't *believe* we do that in any unit tests, but it is a handy thing to lose

Reply:
since jest automatically collects coverage with `--coverage`, are the lcov files written anywhere so we drop the `nyc` calls (for non-smoke tests) altogether?

Reply:
if we use jest everywhere we won't need nyc for the unit tests at all and we can use the lcov/html files they generate in `./coverage`

right now, jest coverage output is just disabled and it lets nyc collect/combine everything since we still need it for the mocha cases